Build.proj

```diff
@@ -142,7 +142,7 @@
       BuildInParallel="true"
       Projects="$(SolutionToBuild)"
       Properties="$(SolutionProperties);$(MSBuildCommonProperties)"
-      Targets="$(InteractiveBuildTarget)"/>
+      Targets="Restore;$(InteractiveBuildTarget)"/>
```

Comment:
Here and below shouldn't have `Restore`, because then you'd be restoring on `Clean`, too. You should it in the `Build` target instead, right? Where we have `Properties="InteractiveBuildTarget=Build"`?

Reply:
Lol nevermind, reading next commit now :-P

Reply:
No, this is fine. Clean goes away in the next commit. Also keep in mind the real goal is nuking Build.proj altogether, so for right now I don't care if it's a little exhaustive, as long as it works in CI.